python/paddle/v2/dataset/tests/common_test.py

```diff
@@ -57,6 +57,22 @@ def test_cluster_file_reader(self):
         for idx, e in enumerate(reader()):
             self.assertEqual(e, str("0"))
 
+    def test_convert(self):
+        def test_reader():
+            def reader():
+                for x in xrange(10):
```

Comment:
I think this test will break when 10 is changed to 4. According to line 191 `if i % num_shards == 0 and i >= num_shards:`